.travis.yml

```diff
@@ -83,7 +83,7 @@ jobs:
         - powershell "$CONDA_PATH; conda install --yes -c msys2 m2-base m2-make m2w64-toolchain libpython"
         - powershell "$CONDA_PATH; pip install tox"
       script:
-        - powershell "$CONDA_PATH; tox -e py3"
+        - powershell "$CONDA_PATH; tox -e pyw3"
```

Comment:
Can you add a comment here explaining why mingw is needed?

Reply:
I actully do not know if it is strictly needed. I did not try VS (doubt will work out of the box). At the moment failed because of `-Werror`

Reply:
Do you have a link to the failure?

Reply:
https://travis-ci.org/themperek/cocotb/jobs/646754559#L454

Reply:
Perhaps we should just set all these warning flags via TOX, so we can pick them per-compiler?